src/ProximalOperators.jl

```diff
@@ -11,6 +11,7 @@ const ArrayOrTuple{R} = Union{
     AbstractArray{C, N} where {C <: RealOrComplex{R}, N},
     TupleOfArrays{R}
 }
+const TransposeOrAdjoint{M} = Union{Transpose{C,M} where C, Adjoint{C,M} where C}
```

Comment:
I see no reason to restrict `C` in this case since it is already restricted (defined) by `eltype(M)`, but I'm open to suggestions. The only case I can see where something might possibly go wrong is if `eltype(A) != eltype(A')`, but in that case I think we are in big trouble already by assuming things on `A'A`, and in worst case we make an unnessesary copy in the type-unstable case.

Reply:
Yeah I think this is fine